lib/ReactViews/Custom/Chart/ChartExpandAndDownloadButtons.tsx

```diff
@@ -70,8 +70,9 @@ class ChartExpandAndDownloadButtons extends React.Component<PropsType> {
             }
           });
         });
-        workbench.add(itemToExpand);
+
         try {
+          workbench.add(itemToExpand);
```

Comment:
`expandItem` isn't an `async` function so you can't catch promise rejections from `workbench.add`. I think `workbench.add` should be outside the the `try` so that it's clear that no exceptions are expected to be thrown from it.

Reply:
👍